### PR TITLE
chore: update drone pipeline - FRONT-1255

### PR DESCRIPTION
**Pull request by planctus**
# PR description

As we know this ticket is just for anticipating changes that we requested to be deployed, basically to keep a reference to the latest pipeline requested.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] I have put the vanilla component as `devDependencies`
- [ ] I have put the specs package as `devDependencies`
- [ ] I have added the components directly used in the twig file (with `include` or `embed`) as `dependencies`
- [ ] My component is listed in `@ecl-twig/ec-components`'s `dependencies`
- [ ] My variables naming follow the guidelines (snake case for twig)
- [ ] I have provided tests
- [ ] I have provided documentation (for the "notes" tab)
- [ ] If my local `yarn.lock` contains changes, I have committed it
- [ ] I have given my PR the proper label (`pr: review needed` to indicate that I'm done and now waiting for a review ,`pr: wip` to indicate that I'm actively working on it ...)

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -21,16 +21,16 @@ pipeline:
...
     group: previews
     image: node:dubnium
     secrets: [gh_token, netlify_auth_token, netlify_site_id]
     commands:
-      - yarn clean
-      - yarn dist
-      - 'yarn netlify deploy --site $${NETLIFY_SITE_ID} --message "Drone build: ${CI_BUILD_NUMBER}"'
+      - yarn twing:clean
+      - yarn twing:dist
+      - 'yarn netlify deploy --site $${NETLIFY_SITE_ID} --alias ${DRONE_COMMIT_BRANCH} --message "Drone build: ${CI_BUILD_NUMBER} for the branch: ${DRONE_COMMIT_BRANCH}"'
```
Comment (kalinchernev) [inline .drone.yml:33]: I will setup drone locally for the ecl-twig project in a bit to have it for future updates too.
Although I haven't tested locally yet, this change looks problematic. This deployment message is used by the script https://github.com/ec-europa/ecl-twig/blob/master/scripts/set-netlify-deployment-status.js#L58 to find the specific deployment. It seems to me the current deployment will not be found and the status will not be updated.

File: .drone.yml [hunk 2]
```diff
@@ -59,10 +59,10 @@ pipeline:
       - npm install -g n && n stable
       - yarn install
       - rm -rf netlify.toml
-      - yarn setupPhp
-      - yarn dist:php
+      - yarn php:install
+      - yarn php:dist
       - yarn diff
-      - 'yarn netlify deploy --dir=php/dist --site $${NETLIFY_SITE_ID} --message "Drone build: ${CI_BUILD_NUMBER}"'
+      - 'yarn netlify deploy --dir=php/dist --site $${NETLIFY_SITE_ID} --alias php-${DRONE_COMMIT_BRANCH} --message "Drone build: ${CI_BUILD_NUMBER} for the branch: ${DRONE_COMMIT_BRANCH}"'
```
Comment (kalinchernev) [inline .drone.yml:65]: Same comment here

**Comment by planctus** [comment]
This pipeline has been already deployed, surprisingly. https://drone.fpfis.eu/ec-europa/ecl-twig/3337 seems to work fine with the new scripts and the solution for differentiating the urls for EC and EU. There is the EU build failing sometimes since we introduced in the drone build, so this is not new.
The url is not working but the guess is that it depends on the uppercase in the branch name,  a lowercase branch name it has been proven working on a different branch.

**Comment by planctus** [comment]
https://github.com/ec-europa/ecl-twig/pull/487, it seems that the scripts is capable to return the status, the js one is not in place due to the failure in the build of EU, i'll restart the build (https://drone.fpfis.eu/ec-europa/ecl-twig/3340/) on drone which usually makes also the EU build work, but that looks pretty randomic.
The url (php one) is reachable:
https://php-front-0000--ecl-twig.netlify.app/

**Comment by kalinchernev** [comment]
` it seems that the scripts is capable to return the status` - return yes, set - no. 
The commit https://github.com/ec-europa/ecl-twig/pull/488#issuecomment-652844239: 
Screenshot from 2020-07-02 10-01-13

**Comment by planctus** [comment]
Yep, you're right, not sure what i've seen before, then. Uff...

**Comment by planctus** [comment]
So, i reverted the changes in the comments, will ask for another deployment, my bad. Thanks for spotting this, it is not in this update that we edited the messages, but in the previous one and it was untested.

**Comment by planctus** [comment]
create this new ticket for multisite: https://citnet.tech.ec.europa.eu/CITnet/jira/browse/MULTISITE-23922

**Comment by kalinchernev** [comment]
Will have to be synched back to develop too

**Pull request merged**